src/ui/components/Rating/styles.scss

```diff
@@ -38,3 +35,25 @@ $star-size: 46px;
     background-image: url('./img/outlined-star.svg');
   }
 }
+
+.Rating-selected-star,
+.Rating--editable .Rating-selected-star {
+  background-image: url('./img/yellow-star.svg');
+}
+
+.Rating--small {
+  $star-size: 13px;
+  $star-gutter: 4px;
+
+  .Rating-star-group {
+    display: flex;
+    min-width: $star-size * 5 + $star-gutter * 4;
```

Comment:
ah, I can totally remove `min-width`, thanks. That was leftover from when I was using floats (for whatever reason).